frontend/src/components/AddPostForm/AddPostForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import {
+  Wrapper,
+  FormTitle,
+  Icon,
+  TitleInput,
+  MessageInput,
+  SubmitInput,
+} from './AddPostForm.styled';
+
+const AddPostForm = () => {
+  const [postObject, setPostObject] = useState({});
```

Comment:
```suggestion
  const [postForm, setPostForm] = useState({});
```
?

Reply:
Hmm, tutaj się coś dziwnego zadziało, bo u mnie jest postObject. Wcześniej miałem postForm, ale postObject bardziej mi pasuje.

Reply:
Jakkolwiek nie powinniśmy używać słowa `Object`, bo w JS wszystko jest obiektem.
Użycie Form, nie będzie błędem, ale możesz użyć innej nazwy jeśli chcesz. Jakkolwiek "object" unikamy

Reply:
Jeśli jest to obiekt danych formularza, to też moglibyśmy użyć "FORM". Jakkolwiej "data" też bym unikał, bo wszystko jest danymi :) "Data" stosujemy zwykle do danych pobranych z API (bez obróbki tych danych).

Reply:
Jeśli są to dane formularza — to po kliknięciu "submit" powiesz, że wysyłasz "formularz" do BE.
Więc taki skrót może być ok.

Pamiętaj też, że możesz wielokrotnie użyć useState, i każdy z nich może mieć dane jednego inputu.  Taką konstrukcję nazwiesz np. `titleInput` i nie musisz pisać tu Data, albo Object. Podobnie ma sie z formularzem, też użyjesz np `comentForm`

Reply:
> pamiętaj też, że `Form` to komponent, a `form` to obiekt z danymi. Wielkość ma znaczenie

Tak, tak, wiem. Zrobiłem z forma komponent, żeby go ostylować. Mam nadzieję, że tak jest ok :)

Reply:
źle chyba się zrozumieliśmy.

Reply:
Nie o tym myślałem, ale już wszystko jasne :)